cli/codegen/generate_vfs.go

```diff
@@ -0,0 +1,72 @@
+// +build ignore
+
+package main
+
+import (
+	"net/http"
+
+	"github.com/apex/log"
+	"github.com/shurcooL/vfsgen"
+	"github.com/tarantool/cartridge-cli/cli/codegen/static"
+)
+
+type generatedTemplate struct {
```

Comment:
Why did you call it template? Maybe it's `generateFSOpts` or smth like this? As for me, it looks like opts.